modules/tibc/light-clients/09-eth/types/client_state.go

```diff
@@ -0,0 +1,256 @@
+package types
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+
+	commitmenttypes "github.com/bianjieai/tibc-go/modules/tibc/core/23-commitment/types"
+	"github.com/bianjieai/tibc-go/modules/tibc/core/exported"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/light"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/trie"
+)
+
+var _ exported.ClientState = (*ClientState)(nil)
+
+func (m ClientState) ClientType() string {
+	return exported.ETH
+}
+
+func (m ClientState) GetLatestHeight() exported.Height {
+	return m.Header.Height
+}
+
+func (m ClientState) Validate() error {
+	return m.Header.ValidateBasic()
+}
+
+func (m ClientState) GetDelayTime() uint64 {
+	return 0
+}
+
+func (m ClientState) GetDelayBlock() uint64 {
+	return 0
```

Comment:
Should it be a non-zero value here?
@chengwenxi

Reply:
clientState.BloclDelay